dashboard/frontend/state/models/runModel.js

```diff
@@ -0,0 +1,32 @@
+import { action, observable, makeObservable } from 'mobx';
+import { socket } from '../../pages/_app';
+
+export default class RunModel {
+    @observable status;
+
+    constructor(runData) {
+        makeObservable(this);
+        this.command = runData.command;
+        this.id = runData.id;
+        this.startTime = runData.startTime;
+        this.status = 'active';
+    }
+
+    @action emitPause() {
+        socket.emit('pause', {
+            id: this.id
+        });
```

Comment:
Future improvement: When emitting an action that has to be handled on the backend, we should disable the button until that action either succeeds, fails or times out to avoid multiple calls for the same action.